apps/ssr-tests-v9/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "target": "ES2019",
+    "allowSyntheticDefaultImports": true,
```

Comment:
Without it `chalk` is broken.

```ts
import chalk from 'chalk';
```

```
src/test.ts:1:8 - error TS1259: Module '"/Users/olfedias/WebstormProjects/office-ui-fabric-react/node_modules/chalk/index"' can only be default-imported using the 'allowSyntheticDefaultImports' flag

1 import chalk from 'chalk';
         ~~~~~

  ../../node_modules/chalk/index.d.ts:415:1
    415 export = chalk;
        ~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'allowSyntheticDefaultImports' flag.
```

---

```ts
import * as chalk from 'chalk';
```

```
/apps/ssr-tests-v9/src/test.ts:68
console.log(chalk.bgRed.whiteBright(' @fluentui/ssr-tests-v9 '));
                        ^
TypeError: Cannot read properties of undefined (reading 'whiteBright')
    at Object.<anonymous> (/apps/ssr-tests-v9/src/test.ts:68:25)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
```

Reply:
I don't mind but I am curious why you get that error ? I tried it out myself in one of `react-menu` test files

![image](https://user-images.githubusercontent.com/20744592/174765872-282e8e31-f49e-46d9-b480-293552b09aae.png)

![image](https://user-images.githubusercontent.com/20744592/174765899-03a6182a-5201-4724-a8b9-ee3d23641e2b.png)

Reply:
yes !